distribution/lib/Standard/Image/0.0.0-dev/src/Codecs.enso

```diff
@@ -40,8 +40,8 @@ read location flags=[] =
         _ -> flags.to_integer
     Panic.recover Any (Image.Image (Java_Codecs.read path read_flags)) . catch e->
         case e of
-            Polyglot_Error _ -> Error.throw (File.Io_Error ('Failed to read the file at ' + path + '.'))
-            err -> Panic.throw err
+            Polyglot_Error _ -> Error.throw (File.Io_Error (File.new path) 'Failed to read the file.')
```

Comment:
`to_display_text` of this will be `Failed to read the file. (filename.ext)` - the placement of the dot is a bit weird. Do you think it is possible to do something about that? Or maybe it's good enough... Very minor nitpick this is :)